src/domain/app/i18n/sv.json

```diff
@@ -1835,6 +1851,10 @@
       "labelSignupExtraInfo": "Ytterligare information (valfritt)",
       "labelStreetAddress": "Gatuadress",
       "labelZipcode": "Postnummer",
+      "notificationSignupDeleted": "Registreringen har raderats",
+      "notificationSignupUpdated": "Deltagarinformation har sparad",
+      "notificationSignupGroupDeleted": "Registreringen har raderats",
```

Comment:
This one also mentions registration although this should be about a signup group

Reply:
Fixed in latest commit